proposals/image_verify_enhancements.md

```diff
@@ -296,7 +299,7 @@ spec:
       - path: "{{ spec.tasks[*] }}"
     imageVerify:
       default: deny
-      digest: true
+      digestVerify: true
```

Comment:
Did you miss `digestMutate` here, or is it intentional?

Reply:
@JimBugwadia ⬆️